Add extra note for open ports (#16698)
DataDog/documentation: content/en/agent/guide/network.md

<div class="alert alert-warning">
All outbound traffic is sent over SSL through TCP / UDP.

> Ensure the Agent is only accessible by your applications or trusted network sources using a firewall rule or similar network restriction. Untrusted access can allow malicious actors to perform several invasive actions, including but not limited to writing traces and metrics to your Datadog account, or obtaining information about your configuration and services.

Reviewer: ```suggestion
<br>
Ensure the Agent is only accessible by your applications or trusted network sources using a firewall rule or similar network restriction. Untrusted access can allow malicious actors to perform several invasive actions, including but not limited to writing traces and metrics to your Datadog account, or obtaining information about your configuration and services.
```

<div class="alert alert-warning">
All outbound traffic is sent over SSL through TCP / UDP.

> <br>

Reviewer: ```suggestion
<br><br>
```

</div>

Open the following ports to benefit from all the **Agent** functionalities:

> <div class="alert alert-warning">
> You should ensure the Agent is only accessible by your applications or trusted network sources using a firewall rule or similar network restriction. Untrusted access could allow malicious actors to perform several invasive actions, including but not limited to writing traces and metrics to your Datadog account, or obtaining information about your configuration and services.
> </div>

Reviewer: ```suggestion
<div class="alert alert-warning">
Ensure the Agent is only accessible by your applications or trusted network sources using a firewall rule or similar network restriction. Untrusted access allows malicious actors to perform several invasive actions, including but not limited to writing traces and metrics to your Datadog account, and obtaining information about your configuration and services.
</div>
```
Also, move this whole section to **before** "Open the following ports..." The current location is interrupts the instruction about which ports to open.
